src/System.Device.Gpio/System/Device/Pwm/Channels/Windows10PwmChannel.Windows.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Threading;
+
+namespace System.Device.Pwm.Channels
+{
+    /// <summary>
+    /// Represents a PWM channel running on Windows 10 IoT.
+    /// </summary>
+    internal class Windows10PwmChannel : PwmChannel
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Windows10PwmChannel"/> class.
+        /// </summary>
+        /// <param name="chip">The PWM chip number.</param>
+        /// <param name="channel">The PWM channel number.</param>
+        /// <param name="frequency">The frequency in hertz.</param>
+        /// <param name="dutyCyclePercentage">The duty cycle percentage represented as a value between 0.0 and 1.0.</param>
+        /// <param name="polarity">The polarity of the PWM signal.</param>
+        public Windows10PwmChannel(
+            int chip,
+            int channel,
+            double frequency,
+            double dutyCyclePercentage = 0.5,
+            PwmPolarity polarity = PwmPolarity.ActiveHigh)
+        {
+            // TODO: This is just a placeholder to complete later.
+        }
+
+        public override double Frequency => throw new NotImplementedException();
```

Comment:
Most of the requested comments on the Unix side are NITs so I think we should be ok to start adding implementaiton of the Windows side. My only big question that could alter API would be whether or not we should remove PwmPolarity class.

Reply:
Thanks for the review/feedback.  I'm still digesting and will tidy up a bit before moving to the Windows flavor.

When it comes to the PwmPolarity enum, I understand it wasn't part of agreed work.  However, after diving into the code/feature, I hope my explanations (https://github.com/dotnet/iot/pull/576#discussion_r300865292 and https://github.com/dotnet/iot/pull/576#discussion_r301008653) would justify on why this is important when it comes to basic electronics.  I can remove if that is what is decided.  It shouldn't take much to remove.

Reply:
@shaggygi I personally think we should not have this API and especially not as part of this work, please create an issue with all the details you put here and then we can decide on that (I'd not add it unless it gets heavily upvoted or have some good justification for it)

Reply:
@krwq understood.  issue #581 created for future discussion.  latest commit removed the PwmPolarity related code.